app/entry.server.tsx

```diff
@@ -1,56 +1,21 @@
-import { PassThrough } from "node:stream";
```

Comment:
🗒️ **Node** streaming fungerer ikke on-the-edge. Har resatt denne filen til default.